packages/atomic/src/components/ipx/atomic-ipx-button/atomic-ipx-button.tsx

```diff
@@ -32,7 +32,7 @@ export class AtomicIPXButton {
   /**
    * Whether the IPX layout is open.
    */
-  private isOpen = false;
+  private isOpen = this.host.classList.contains('btn-open');
```

Comment:
❗️we don't want to expose classes outside the shadowDOM, and it looks like this would be exposed. Additionally, it's not prefixed with `atomic-` and uses a fairly generic name, therefore may be affected by other styling on the implementer's page.

I would recommend replacing this with a prop that has `mutable: true` and `reflect: true`, which will have the same effect without the consequences, and will also remove the need for `classList.add`/`classList.remove` in the code.